testproject/testapp/tests/test_token_create.py

```diff
@@ -1,78 +1,71 @@
+import pytest
 from django.contrib.auth import user_logged_in, user_login_failed
-from djet import assertions
 from rest_framework import status
 from rest_framework.reverse import reverse
-from rest_framework.test import APITestCase
-from testapp.tests.common import create_user
 
 from djoser.conf import settings
 
 
-class TokenCreateViewTest(
-    APITestCase,
-    assertions.StatusCodeAssertionsMixin,
-    assertions.InstanceAssertionsMixin,
-):
-    def setUp(self):
-        self.signal_sent = False
-        self.base_url = reverse("login")
+@pytest.mark.django_db
+class TestTokenCreateView:
```

Comment:
I'm not sure why there is some inconsistency when it comes to grouping tests in classes like here, or not like in e.g. `test_password_reset.py`.